### PR TITLE
Custom CSS loader for JEvents and modal css fix

### PR DESCRIPTION
**Pull request by carcam**

### CODE REVIEW

File: libraries/jevmodal/media/css/jevmodal.css [hunk 1]
```diff
@@ -161,7 +162,7 @@
 	max-height:100%;
 }
 .jevmodal .modal-dialog {
-    height: 100%;
+    height: 95%;
```
Comment (GeraintEdwards) [inline libraries/jevmodal/media/css/jevmodal.css]: Is this to deal with the scrollbar issue?  I've dealt with this before and thought I had a better solution - from memory its usually a padding or margin issue and using the height CSS won't help on small screens .  Is there a URL where you can see the problem in action?

Comment (carcam) [inline libraries/jevmodal/media/css/jevmodal.css]: It's indeed! The site is Kenneth's testing site, let me know if you need site access.

Comment (GeraintEdwards) [inline libraries/jevmodal/media/css/jevmodal.css]: Please send me the details and I'll take a look.  Did you implement 
these changes there already?

Geraint

Comment (carcam) [inline libraries/jevmodal/media/css/jevmodal.css]: Actually Kenneth did in any of his custom css files and told me, but you will be able to find it easily with CSS inspector. I did not apply any changes on his JEvents files as he already took care of it.

Access details will be in your inbox in 5 minutes.

**Comment by GeraintEdwards** [comment]
Did you see my fix for Kenneth's site?  I don't think there is now the need for this change

**Comment by carcam** [comment]
There are also other changes that I pushed into my devel branch.

**Pull request merged**